part-spec/usbc.json

```diff
@@ -0,0 +1,184 @@
+{
+    "$id": "https://github.com/chromeos/digital-datasheets/blob/main/part-spec/usbc.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "usbc",
+    "oneOf": [
+        {
+            "title": "usb-c pd controller",
+            "type": "object",
+            "required": [
+                "componentID",
+                "pdVersion"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "pdVersion": {
+                    "description": "version of power delivery spec implemented by controller",
+                    "examples": [
+                        "pd2",
+                        "pd3"
+                    ],
+                    "type": "string"
+                },
+                "usbTypecRevision": {
+                    "description": "usb type-c spec revision implemented by controller",
+                    "examples": [
+                         "1.2",
+                         "1.3"
+                    ],
+                    "type": "string"
+                },
+                "roleSupported": {
+                    "description": "roles supported by pd controller",
+                    "examples": [
+                        "Source",
+                        "Sink",
+                        "DRP"
+                     ],
+                     "type": "string"
+                },
+                "fastRoleSwapSupport": {
+                    "description": "whether the pd controller supports fast role swap (FRS)",
```

Comment:
Would this need to be broken out as FRS support for initial sink versus initial source?  Or is the assumption that a part supporting one will also support the other?